### ISSUE TITLE
About page for data repository

### ISSUE THREAD

**Issue by MortenHofft** [gbif/portal16#637]
For the data repository we need a tool item in Contentful with a description of what it is (the relation to dataone and datacite?) and perhaps how to use it. This will go at https://www.gbif-uat.org/data-repository/about

**Comment by dnoesgaard** [comment]
Although I don't have the content ready, here's the tool item:

33ZlEJM58I4iMqSqCiiQgg

What will the official url of the tool be? Right now it's /data-repository but we've used /tools/xxx for the other "GBIF" tools...

**Comment by MortenHofft** [comment]
Good point. I'm thinking that perhaps we shouldn't label it as a tool, but simply our repository?
Like we have `occurrence/download` we have `data-repository/upload`

**Comment by dnoesgaard** [comment]
Would it be part of the tools menu? ;)

**Comment by MortenHofft** [comment]
it already is if you have the right role. Argh - i don't know

**Comment by MortenHofft** [comment]
So is the IPT https://www.gbif.org/ipt

**Comment by dnoesgaard** [comment]
Either way is fine with me :)

**Comment by MortenHofft** [comment]
I'm in favour of a no tools URL. It is my impression that we might want this to be a prominent thing going forward so I'd rather have a simpler url.. And urls and menu position is largely disconnected anyhow for other menu items. If better arguments come along I will move it.

**Comment by dnoesgaard** [comment]
The tool item is now showing in the about tab: https://www.gbif.org/data-repository/about

I'd love to write this but I'd probably need a little background on what this thing is actually for. Who do I talk to?

**Comment by MortenHofft** [comment]
@dnoesgaard 
> Who do I talk to?

to @fmendezh who know the scope of the project best
